internal/auth/scopes.go

```diff
@@ -0,0 +1,33 @@
+package auth
+
+var RequiredScopes = []string{
+	"openid",
+	"offline_access", // for retrieving refresh token
+	"create:clients", "delete:clients", "read:clients", "update:clients",
+	"create:resource_servers", "delete:resource_servers", "read:resource_servers", "update:resource_servers",
+	"create:roles", "delete:roles", "read:roles", "update:roles",
+	"create:rules", "delete:rules", "read:rules", "update:rules",
+	"create:users", "delete:users", "read:users", "update:users",
+	"read:branding", "update:branding",
+	"read:email_templates", "update:email_templates",
+	"read:connections", "update:connections",
+	"read:client_keys", "read:logs", "read:tenant_settings",
+	"read:custom_domains", "create:custom_domains", "update:custom_domains", "delete:custom_domains",
+	"read:anomaly_blocks", "delete:anomaly_blocks",
+	"create:log_streams", "delete:log_streams", "read:log_streams", "update:log_streams",
+	"create:actions", "delete:actions", "read:actions", "update:actions",
+	"create:organizations", "delete:organizations", "read:organizations", "update:organizations", "read:organization_members", "read:organization_member_roles",
+	"read:prompts", "update:prompts",
+	"read:attack_protection", "update:attack_protection",
+}
+
+// RequiredScopesMin returns minimum scopes used for login in integration tests.
```

Comment:
This comment is no longer accurate, as `RequiredScopesMin()` is only used in `GetAccessTokenFromClientCreds()`.

Reply:
Yes, this is a good suggestion. Done!